client/src/app/components/album-detail/album-detail.component.css

```diff
@@ -5,30 +5,35 @@
 }
 
 .items {
-  display: flex;
+  display:flex;
   flex-wrap: wrap;
 }
 
 
 .upload-item {
-  min-width: calc(100% - 6px);
-  min-height: 100%;
+  width: calc(100% - 3%);
+  margin-top: 3%;
+  margin-bottom: 3%;
+  margin-left:1.5%;
   background-color: #d7d9df;
+  border-radius: 20px;
+
 }
 
-.item {
-  max-width: calc(33.33% - 6px);
-  margin: 20px 3px 4px 3px;
-  display: flex;
-  flex-direction: column;
-  align-content: center;
-  justify-content: center;
-  border-radius: 10px;
-  padding: 6px;
+.col {
+   flex: 0 0 calc(44%);   /* flex-basis adjustment for margin */
+   margin: 3% 2.9% 3% 3%;
```

Comment:
i really recommend using pixels for margins and padding % is like asking for trouble - try viewing page on a mobile device this will not work well